dabl-meta.yaml

```diff
@@ -4,13 +4,14 @@
 catalog:
     name: dabl-integration-slack
     group_id: com.digitalasset
-    version: 0.5.1
-    short_description: Slack Integration
+    version: 0.5.2
+    short_description: Slack
     description: Slack Send and Receive
     release_date: 2020-08-13
```

Comment:
Does the release date need to change?

Reply:
No. It's not displayed and about to be removed.